### PR TITLE
[4.1] Private Messages Emails

### PR DESCRIPTION
**Pull request by brianteeman**
PR for #29533 (and more)

Since the component was changed to include the actual message in the email it is no longer a notification so I updated the description show in the mail templates component

image

### Before this PR the subject of the email that was generated was
**`New Message from <user> at <sitename>`**

The actual subject of the message was included in the body of the email.
"New Message from <user>" is not needed. We know who the email is from in the usual way and our inbox will show if it is new or not ;)

This also meant that you would have an inbox with multiple messages that all had the exact same title :(

image

### With this PR the subject of the email that is generated is
**`[<sitename>] - Subject`**
and the body of the email only contains the body of the message not subject and body.

image

_The messages can of course still be managed and updated with the mail templates component. This is just setting a better default._

### CODE REVIEW

**Comment by tecpromotion** [comment]
If the name of the website (sitename) is long, there is a risk that the subject will get lost in the background.
The idea is great but maybe you should turn it around? What do you think @brianteeman ?

**Comment by brianteeman** [comment]
I definitely want the sitename to be first for the exact scenario you suggest. If sitea is more important to me than siteb then I want to be sure that I can see which site it is in the message.

But as its configurable now then a user could change it as they see fit.

**Review by tecpromotion — APPROVED** [review]
thanks @brianteeman

**Comment by tecpromotion** [comment]
I have tested this item :white_check_mark: successfully on 7be569e3a5080f38243a633f8532c599b43ce703<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37133">issues.joomla.org/tracker/joomla-cms/37133</a>.</sub>

File: administrator/language/en-GB/com_messages.ini [hunk 1]
```diff
@@ -58,8 +58,8 @@ COM_MESSAGES_N_ITEMS_TRASHED="%d messages trashed."
 COM_MESSAGES_N_ITEMS_TRASHED_1="Message trashed."
 COM_MESSAGES_N_ITEMS_UNPUBLISHED="%d messages marked as unread."
 COM_MESSAGES_N_ITEMS_UNPUBLISHED_1="Message marked as unread."
-COM_MESSAGES_NEW_MESSAGE="New Message from {FROMNAME} at {SITENAME}"
-COM_MESSAGES_NEW_MESSAGE_BODY="{SUBJECT} \n\n{MESSAGE}"
+COM_MESSAGES_NEW_MESSAGE="[{SITENAME}] - {SUBJECT}"
```
Comment (Quy) [inline administrator/language/en-GB/com_messages.ini:61]: Is the hyphen necessary? The [] acts as a separator.

Comment (brianteeman) [reply administrator/language/en-GB/com_messages.ini:61]: Personal taste.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 7be569e3a5080f38243a633f8532c599b43ce703<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37133">issues.joomla.org/tracker/joomla-cms/37133</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37133">issues.joomla.org/tracker/joomla-cms/37133</a>.</sub>

**Comment by HLeithner** [comment]
Thanks

**Comment by brianteeman** [comment]
thanks

**Pull request merged**